drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -190,6 +190,15 @@ static int sdw_compute_group_params(struct sdw_bus *bus,
 	}
 
 	list_for_each_entry(m_rt, &bus->m_rt_list, bus_node) {
+		if (m_rt->stream == stream) {
+			if (stream->state != SDW_STREAM_CONFIGURED)
+				continue;
+		} else {
+			if (m_rt->stream->state != SDW_STREAM_ENABLED &&
```

Comment:
and this is not intuitive at all. What are you trying to say here? Ignore a runtime thats not associated with the stream if it has already been activated once?

Reply:
Yes, DISABLED is about paused. Please see https://github.com/thesofproject/linux/blob/topic/sof-dev/Documentation/driver-api/soundwire/stream.rst#soundwire-stream-states and https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sdw_utils/soc_sdw_utils.c#L809